### PR TITLE
Adding snippets for Chat token management

### PR DESCRIPTION
**Pull request by dr0nius**
Particularly addressing three scenarios:
- initializing the client with initial token
- updating the token with provided string
- using the built-in token lifecycle events

### CODE REVIEW

File: ip-messaging/tokens/initialization/initialization.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import com.twilio.chat.ChatClient;
+import com.twilio.chat.ChatClientListener;
+
+ChatClient.Properties.Builder builder = new ChatClient.Properties.Builder();
+ChatClient.Properties properties = builder.createProperties();
+// See below for event listener implementation
+ChatClientListener listener = new ChatClientListener();
```
Comment (berkus) [inline ip-messaging/tokens/initialization/initialization.java]: `ChatClientListener` is an interface, you can't create an instance.

Comment (berkus) [reply ip-messaging/tokens/initialization/initialization.java]: Use `Listener` probably, as this is what is used below in the examples.

Comment (dr0nius) [reply ip-messaging/tokens/initialization/initialization.java]: Ouch, facepalm

**Comment by berkus** [comment]
👍

**Pull request merged**